models/Movie.model.js

```diff
@@ -0,0 +1,18 @@
+const { mongoose, Schema, model } = require('mongoose');
```

Comment:
Here we would remove mongoose from the destructuring

```suggestion
const  mongoose, { Schema, model } = require('mongoose');
```